src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinatorViewModel.swift

```diff
@@ -8,13 +8,18 @@ import PDFKit
 import OpenCombine
 
 enum FileScannerError: CaseIterable {
+	static var allCases: [FileScannerError] {
+		return [.noQRCodeFound, .fileNotReadable, .invalidQRCode, .photoAccess, .passwordInput, .unlockPDF, .alreadyRegistered, .qrCodeParserError(.scanningError(.codeNotFound))]
+	}
```

Comment:
if added manually you no longer need to fulfill `CaseIterable`

Reply:
I think you could also just remove the whole CaseIterable conformance and this manual implementation because I think this is just for a unit that we don't really need